src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/DoNotUseInsecureXSLTScriptExecution.cs

```diff
@@ -35,11 +35,9 @@ public sealed class DoNotUseInsecureXSLTScriptExecutionAnalyzer : DiagnosticAnal
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(RuleDoNotUseInsecureXSLTScriptExecution);
 
-#pragma warning disable RS1026 // Enable concurrent execution
         public override void Initialize(AnalysisContext context)
-#pragma warning restore RS1026 // Enable concurrent execution
         {
-            //analysisContext.EnableConcurrentExecution();
+            context.EnableConcurrentExecution();
```

Comment:
So it is safe to enable it now? (The reference in the issue had a comment `// TODO: Make analyzer thread-safe.` Could not find from history how its removed or who removed it

Reply:
@buyaa-n I'm not entirely sure tbh. As mentioned in PR description:

> Note: I don't see any concurrency issues in the implementation, but this needs to be carefully confirmed.

cc @sharwell